strongbox-security/strongbox-authentication-registry/src/main/java/org/carlspring/strongbox/authentication/DatabaseExternalUsersCacheManager.java

```diff
@@ -56,7 +56,7 @@ public User cacheExternalUserDetails(String sourceId,
             userEntry.setEnabled(user.isEnabled());
             userEntry.setRoles(user.getRoles());
             userEntry.setSecurityTokenKey(user.getSecurityTokenKey());
-            userEntry.setLastUpdated(new Date());
+            userEntry.setLastUpdated(LocalDateTime.now());
```

Comment:
So, Janusgraph+Cassandra, unlike OrientDB, supports `LocalDateTime`? Cool! :)

Reply:
// todo: shouldn't we use `UTC` for storing date times in the database? If you guys agree, we can make a follow-up task :)

Reply:
> So, Janusgraph+Cassandra, unlike OrientDB, supports LocalDateTime? Cool! :)

they don't :) it soted as `Long` timestamp

Reply:
> shouldn't we use UTC for storing date times in the database? If you guys agree, we can make a follow-up task :)

+1

Reply:
@sbespalov : I'm not sure I understand your reaction. Do we need a follow-up task for this, or what...? If so, could you please create one...?

Reply:
will do

Reply:
[issues/1845](https://github.com/strongbox/strongbox/issues/1845)

Reply:
Thanks for raising this!